content/updates/terraform-null-label.md

```diff
@@ -0,0 +1,194 @@
+
+Now, obviously that’s a very simplified and abstract example. In complex configurations, there are likely going to be many resources and modules with different usage, and in this case even with `terraform-null-label` it can start to get cumbersome to have to define new label modules each time you need to have variations in your ID elements. There is a solution to that however as one of the module’s most powerful features addresses exactly that: the `context` input + output value.
+
+Going back to our example, what if we needed to deploy another internal load balancer alongside the public one? That is going to share a lot of the same label id elements with a couple of overrides. Here’s how easy that is:
+
+```hcl
+  module "public_alb_label" {
+    source  = "cloudposse/label/null"
+    version = "0.25.0"
+
+    namespace   = "mp"
+    stage       = "prod"
+    environment = "uw2"
+    name        = "alb"
+    attributes  = ["public"]
+
+    tags = {
+      BusinessUnit       = "ENG"
+      Team               = "Reporting"
+      ManagedByTerraform = "True"
+    }
+  }
+
+  module "private_alb_label" {
+    source  = "cloudposse/label/null"
+    version = "0.25.0"
+
+    attributes  = ["private"]
+
+    # Here's the important bit
+    context = module.public_alb_label.context
+  }
+```
+
+By passing in the context output from the first label module, all of the original values are present, with only the defined ones overridden. With that, you get:
+
+* `module.public_alb_label.id` equates to `mp-uw2-prod-alb-public`
+* `module.public_alb_label.tags` equates to:
+
+   ```yaml
+   Namespace: "mp"
+   Environment: "uw2"
+   Stage: "prod"
+   Name: "alb"
+   Attributes: "public"
+   BusinessUnit: "ENG"
+   Team: "Reporting"
+   ManagedByTerraform: "True"
+
+* `module.private_alb_label.id` equates to `mp-uw2-prod-alb-private`
+* `module.private_alb_label.tags` equates to:
+
+   ```yaml
+   Namespace: "mp"
+   Environment: "uw2"
+   Stage: "prod"
+   Name: "alb"
+   Attributes: "private"
+   BusinessUnit: "ENG"
+   Team: "Reporting"
+   ManagedByTerraform: "True"
+
+The new label module only needs to have new values defined; the previous values are still present and do not need to be redefined.
+
+Admittedly, that’s still a pretty simplified example. Real Terraform configurations often employ root and child modules and various workspaces / folders for each; to this point our examples have been constrained to one workspace, working directory, and statefile. There is a nice solution to simplify that process however via the `context.tf` file, but we’re going to explore that advanced usage in a separate article, so be sure to watch this space!
```

Comment:
It's probably worth to add a link to https://github.com/cloudposse/terraform-null-label/blob/main/exports/context.tf file for some context (hey, it's a wordplay!). At least to provide some understanding that it's a file that lives within this module for those who a new to it.